src/unity/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -832,30 +832,33 @@ def export_coreml(self, path, image_shape=(256, 256)):
         coremltools.utils.rename_feature(spec,
                 'transformer__mulscalar0_output', stylized_image, True, True)
 
-        # Support flexible shape
-        flexible_shape_utils = _mxnet_converter._coremltools.models.neural_network.flexible_shape_utils
-        img_size_ranges = flexible_shape_utils.NeuralNetworkImageSizeRange()
-        img_size_ranges.add_height_range((64, -1))
-        img_size_ranges.add_width_range((64, -1))
-        flexible_shape_utils.update_image_size_range(spec, feature_name=self.content_feature, size_range=img_size_ranges)
-        flexible_shape_utils.update_image_size_range(spec, feature_name=stylized_image, size_range=img_size_ranges)
+        if _mac_ver() >= (10, 14):
```

Comment:
Let's think more about this -- I think instead of hardcoding it based on macOS version (and implicitly then Linux drops flexible shape support!), we should take an optional parameter to the export_coreml method, and behave consistently across operating systems.